### PR TITLE
Hide dpad for Shared Sprite Lab projects in mobile [test all browsers]

### PR DESCRIPTION
**Pull request by katleiahramos**
SpriteLab projects in share mode were showing a dpad (inherited from GameLabs). Since SpriteLabs don't utilize dpads (arrow keys only) this change is to hide it by default. 

## Links
- jira ticket: [STAR-1574](https://codedotorg.atlassian.net/browse/STAR-1574)

## Testing story
UI test added

## PR Checklist:

- [x] Tests provide adequate coverage
- [x] Code is well-commented
- [x] User impact is well-understood and desirable
- [x] Pull Request is labeled appropriately

### CODE REVIEW

File: dashboard/test/ui/features/star_labs/share_buttons.feature [hunk 1]
```diff
@@ -17,3 +17,19 @@ Feature: Share Buttons
...
+  # Making DPad button show up in Game Lab and not in Sprite Lab
+
+  Scenario: Dpad does not appear for Sprite Lab Share
+    Given I am on "http://studio.code.org/projects/spritelab/"
+    And I wait for the page to fully load
+    When I navigate to the shared version of my project
+    And I wait until element "#gameButtons" is visible
+    And element "#studio-dpad-rim" is not displayed
+  
+  Scenario: Dpad appears for Sprite Lab Share
```
Comment (ajpal) [inline dashboard/test/ui/features/star_labs/share_buttons.feature]: nit: Game Lab

**Review by ajpal — APPROVED** [review]
lgtm! thanks for adding a test too!

File: dashboard/test/ui/features/star_labs/share_buttons.feature (hunk 1 above)
Comment (katleiahramos) [reply dashboard/test/ui/features/star_labs/share_buttons.feature]: Thanks for catching this! updated here: https://github.com/code-dot-org/code-dot-org/commit/bb53155ce9b52ee6445b5c8b13ce40f57e766417

**Comment by epeach** [comment]
First PR yay!
yaydance

**Pull request merged**